pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/creator/SegmentGeneratorConfig.java

```diff
@@ -784,6 +786,14 @@ public void setNullHandlingEnabled(boolean nullHandlingEnabled) {
     _nullHandlingEnabled = nullHandlingEnabled;
   }
 
+  public boolean useDefaultValueOnError() {
```

Comment:
this looks like not used. but anyway, might want to change its name to isUseDefaultValueOnError()

Reply:
done